warehouse/forklift/legacy.py

```diff
@@ -1315,6 +1316,9 @@ def file_upload(request):
                 },
             )
 
+    metrics = request.find_service(IMetricsService)
```

Comment:
load this at open, emit a `warehouse.upload.attempt` metric. this will allow us to look for failure count before more fine grained metrics are implemented.